modules/core/client/components/TrEditor.js

```diff
@@ -0,0 +1,202 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import MediumEditor from 'react-medium-editor';
```

Comment:
Heh, updated last time 4 years ago.... https://www.npmjs.com/package/react-medium-editor

We might need to look for an alternative in a couple years. ;-)

Reply:
Indeed, and it also uses bits of react that are deprecated and will be removed in the future.

Fortunately, it's only a relatively small shim between react and medium-editor, so not too much to reimplement ourselves if needed.

Also has had a few commits more recently https://github.com/wangzuo/react-medium-editor/commits/master (just no release).

They actually recommend to use https://draftjs.org/ instead, which could be worth investigating.

Reply:
Yepp, we can cross that bridge once the time comes.